### PR TITLE
 Add python 3.8 to the build matrix

### PR DESCRIPTION
**Pull request by hmaarrfk**
## Description

Lets see how this works

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,19 +69,16 @@ matrix:
         - xvfb
     - os: linux
       python: 3.7
-      stage: Initial tests
```
Comment (hmaarrfk) [inline .travis.yml]: this will be reverted

**Comment by emmanuelle** [comment]
Great! @hmaarrfk I guess this is not WIP (see title) any more and can be merged?

**Comment by hmaarrfk** [comment]
I don't particularly like adding pre to the test matrix. But we can merge anyway if you like

**Comment by hmaarrfk** [comment]
Especially today that scikit learn release 0.22

**Comment by hmaarrfk** [comment]
now that scikit-learn has published wheels this should pass

**Comment by stefanv** [comment]
Thanks, Mark!

**Comment by hmaarrfk** [comment]
Glad to help out!

**Pull request merged**